### PR TITLE
Add ajax parameter for ajax call

### PR DESCRIPTION
**Pull request by luigimassa**
I added a small specification when the call of the controller is done via ajax

### CODE REVIEW

File: src/content/1.7/modules/concepts/controllers/front-controllers.md [hunk 1]
```diff
@@ -166,6 +166,17 @@ secured by redirecting it to HTTPS.
...
+
+When you call controller via ajax, you nedd to add ajax parameter in teh url.
+
+```php
+Context::getContext()->link->getModuleLink('cheque', 'validation', array('idPayment' => 1337, 'ajax'=>true);
+```
+
+* Without URL rewriting: `http://<shop_domain>/index.php?idPayment=1337&fc=module&module=cheque&controller=validation&id_lang=1&ajax=true`
+* With URL rewriting: `http://<shop_domain>/en/module/cheque/validation?idPayment=1337&ajax=true`
+
```
Comment (Quetzacoalt91) [inline src/content/1.7/modules/concepts/controllers/front-controllers.md:179]: What do you think about moving this part between `### Example of method calls` and `## Restricting access`? We could bring these examples together.

Comment (luigimassa) [reply src/content/1.7/modules/concepts/controllers/front-controllers.md:179]: I think it's a good idea. In ### Example of method calls is best.

**Comment by Progi1984** [comment]
Ping @luigimassa

File: src/content/1.7/modules/concepts/controllers/front-controllers.md [hunk 2]
```diff
@@ -137,6 +137,17 @@ Context::getContext()->link->getModuleLink('cheque', 'validation', array('idPaym
 * Without URL rewriting: `http://<shop_domain>/index.php?idPayment=1337&fc=module&module=cheque&controller=validation&id_lang=1`
 * With URL rewriting: `http://<shop_domain>/en/module/cheque/validation?idPayment=1337`
 
+### Ajax request
+
+When you call controller via ajax, you nedd to add ajax parameter in teh url.
```
Comment (Progi1984) [inline src/content/1.7/modules/concepts/controllers/front-controllers.md:142]:
```suggestion
When you call controller via AJAX, you nedd to add `ajax` parameter in the url.
```

**Comment by matks** [comment]
Thank you @luigimassa

**Pull request merged**